### ISSUE TITLE
Yazıcı Şablonumuza {DEPARTMAN} kodu eklemeliyiz.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ahmet328/sambapos#36]
```
Yazıcı Şablonumuza {DEPARTMAN} kodu eklemeliyiz.
```

Original issue reported on code.google.com by `isaakkay...@gmail.com` on 23 Sep 2011 at 11:27

**Comment by GoogleCodeExporter** [comment]
Original comment by `emree...@gmail.com` on 24 Sep 2011 at 1:34
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `emree...@gmail.com` on 24 Sep 2011 at 3:03
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2a1bade5b724.
```

Original comment by `emree...@gmail.com` on 28 Sep 2011 at 4:39
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Kontrol Edildi
```

Original comment by `mehmetak...@gmail.com` on 4 Oct 2011 at 12:06
- Changed state: **Verified**
